examples/controls/src/app.js

```diff
@@ -3,15 +3,11 @@ import React, {Component} from 'react';
 import {render} from 'react-dom';
 import MapGL, {Marker, Popup, NavigationControl} from 'react-map-gl';
 
-import CityPin from './components/city-pin';
+import CityPin from './city-pin';
 
-import CITIES from './data/cities.json';
+import CITIES from './cities.json';
 
-import './app.css';
-import 'mapbox-gl/dist/mapbox-gl.css';
-
-const token = process.env.MAPBOX_ACCESS_TOKEN || // eslint-disable-line
-  'Set MAPBOX_ACCESS_TOKEN environment variable or put your token here.';
+const token = process.env.MapboxAccessToken; // eslint-disable-line
```

Comment:
I should have provided the reference. here it is: https://github.com/uber/react-map-gl/blob/master/src/utils/access-token.js#L15